entgql/extension.go

```diff
@@ -139,6 +139,22 @@ func WithWhereInputs(b bool) ExtensionOption {
 	}
 }
 
+// WithClientNode configures the extension to either add or
+// remove the ClientNodeTemplate from the code generation templates.
+//
+// The ClientNodeTemplate generates client.Node() API for all types
+func WithClientNode(b bool) ExtensionOption {
```

Comment:
```suggestion
// WithClientNode configures the extension to either add or
// remove the ClientNodeTemplate from the code generation templates.
//
// The ClientNodeTemplate generates client.Node() API for all types

// WithNodeDescriptor configures the extension to either add or
// remove the NodeDescriptorTemplate from the code generation templates.
//
// In case this option is enabled, EntGQL generates a `Node()` method for each
// type that returns its representation in one standard way. A common use case for
// this option is to develop an administrator tool on top of Ent as implemented in:
// https://github.com/ent/ent/issues/1000#issuecomment-735663175.
func WithNodeDescriptor(b bool) ExtensionOption {
```